optimum/intel/openvino/quantization.py

```diff
@@ -705,15 +710,16 @@ def _prepare_text_generation_dataset(
 
         num_samples = quantization_config.num_samples or 200
 
-        self.model.request = InferRequestWrapper(self.model.model.request, collected_inputs)
```

Comment:
Thanks for the catch! Could you please remove the `#TODO` above?

Reply:
nice catch removed in https://github.com/huggingface/optimum-intel/pull/727/commits/6e2d448bcdea45678f7e72b868e44757fb262bd9